modules/tensor_mechanics/examples/coal_mining/cosserat_elastic.i

```diff
@@ -0,0 +1,569 @@
+# Strata deformation and fracturing around a coal mine
+#
+# A 2D geometry is used that simulates a transverse section of
+# the coal mine.  The model is actually 3D, but the "x"
+# dimension is only 10m long, meshed with 1 element, and
```

Comment:
I also need to set the stress in the "third" direction and keep the strain in that direction fixed to zero.  Well, i guess i don't **need** to, but it's instructive if i do.